.bazelrc

```diff
@@ -18,7 +18,7 @@ build --ios_simulator_device="iPhone 13"
 build --ios_simulator_version=16.0
 build --verbose_failures
 build --workspace_status_command=envoy/bazel/get_workspace_status
-build --xcode_version=14.0
+build --xcode_version=14.1
```

Comment:
We don't have Xcode 14.1 installed on CI, only Xcode 14.0

Reply:
Bah! Sorry, didn't mean to git push that.